### ISSUE TITLE
Update to 0.11.4 fails

### ISSUE THREAD

**Issue by wilkovanderveen** [ipfs-shipyard/ipfs-desktop#1562]
Please describe what you were doing when this error happened.

**Specifications**

- **OS**: win32
- **IPFS Desktop Version**: 0.11.3
- **Electron Version**: 8.2.5
- **Chrome Version**: 80.0.3987.165

**Error**

```
TypeError: stopIpfs is not a function
    at quitAndInstall (F:\Utilities\IPFS\IPFS Desktop\resources\app.asar\src\auto-updater\quit-and-install.js:13:24)
    at Immediate.<anonymous> (F:\Utilities\IPFS\IPFS Desktop\resources\app.asar\src\auto-updater\index.js:87:9)
    at processImmediate (internal/timers.js:439:21)
```

**Comment by jessicaschilling** [comment]
@rafaelramalho19 Is this anything you've seen in other update issues?

**Comment by lidel** [comment]
This was fixed in [v0.13.2](https://github.com/ipfs-shipyard/ipfs-desktop/releases/tag/v0.13.2), we now use upstream functionality for autoupdates on windows. Please upgrade.
